Kickstarter-iOS/ViewModels/SettingsAccountViewModel.swift

```diff
@@ -43,7 +43,7 @@ public final class SettingsAccountViewModel: SettingsAccountViewModelInputs,
     )
     .switchMap { _ in
       AppEnvironment.current.apiService
-        .fetchGraphUserAccountFields(query: UserQueries.account.query)
```

Comment:
`fetchGraphUser` is now the only method we need to call to get any of the various details associated with the Graphser. Including but not limited to `storedCards`, `account` or `email`.